build/source/driver/multi_driver.f90

```diff
@@ -266,7 +288,7 @@ program multi_driver
 ! nGRU-is the total number of GRUs of the simulation domain
 ! nHRU-is the total number of HRUs of the simulation domain
 ! hruCount-is a local variable for the total number of HRUs in a GRU
-call allocate_gru_struc(nGRU,nHRU,err,message); call handle_err(err,message)
+call allocate_gru_struc(nGRU,nHRU,strtHRU,err,message); call handle_err(err,message)
```

Comment:
The mapping between the HRUs and GRUs is read in `allocatespace.f90` because this is necessary to define the space required for the local HRUs (i.e., the local HRUs within each GRU).